unittests/test-contracts/svnn_ibc/bitset.hpp

```diff
@@ -0,0 +1,42 @@
+using namespace eosio;
+
+class bitset {
+public:
+    bitset(size_t size) 
+        : num_bits(size), data((size + 63) / 64) {}
+    
+   bitset(size_t size, const std::vector<uint64_t>& raw_bitset)
+       : num_bits(size), data(raw_bitset) {}

```

Comment:
might want to consider validating that `raw_bitset` is properly sized. otherwise it seems like it's possible to construct a `bitset` where `num_bits` is greater than actual `data`.

Reply:
Done